README.md

```diff
@@ -154,4 +154,5 @@ Summary
   - or install from [crates.io](https://crates.io/crates/shotgun):
     `cargo install shotgun`
 - [Arch Linux](https://www.archlinux.org/packages/?name=shotgun): `pacman -S shotgun`
+- [Nix/NixOS from 20.03 onwards](https://github.com/NixOS/nixpkgs-channels/blob/nixos-20.03/pkgs/tools/graphics/shotgun/default.nix) `nix-env -iA nixpkgs.shotgun`
```

Comment:
I think I'd rather point to unstable so the link is always to the most current version of the package, and I don't think it's necessary to specify which nixpkgs releases it's available in. Also, `nix-env` isn't really the recommended practice for "installing" packages on NixOS, I guess just `nixpkgs.shotgun` would be enough to get the point across.
```suggestion
- [Nix/NixOS](https://github.com/NixOS/nixpkgs-channels/blob/nixos-unstable/pkgs/tools/graphics/shotgun/default.nix): `nixpkgs.shotgun`
```